src/env.rs

```diff
@@ -52,7 +52,13 @@ pub fn sub_remaining_point(env: &Env, points: u64) -> anyhow::Result<()> {
     Ok(())
 }
 
-/// if get_string throws an exception abort for some reason is being called
+/// Called by the instance when an error popped. It print the filename where the error
+/// had pop up, an error message and more stacktrace information as line and column
+///
+/// This function is automatically exported by AssemblyScript on build and allow us to
+/// log what appened when a smartcontract crashed inside the instance.
+///
+/// To work has to be in the ImportObject in the "env" namespace.
```

Comment:
is a word missing here ? I find this sentence confusing

Reply:
I changed it! :+1: